core/src/main/java/org/apache/iceberg/UnboundBaseFileScanTask.java

```diff
@@ -21,13 +21,15 @@
 import org.apache.iceberg.expressions.Expression;
 import org.apache.iceberg.expressions.ResidualEvaluator;
 
-public class UnboundBaseFileScanTask extends BaseFileScanTask {
```

Comment:
this is important, we should double check we're not making classes unnecessarily public. I cleaned up this case, but let's double check as we iterate.

Reply:
ack